test/unit/controllers/InvitationController.test.js

```diff
@@ -73,7 +73,7 @@ describe('InvitationController', () => {
       })
     })
 
-    it('sends invitations', function () {
+    it('sends invitations', () => {
```

Comment:
we do use ES6 function syntax wherever possible -- in this case we used the traditional syntax so that `this` in `this.timeout` would resolve correctly